--- CYGWIN-PATCHES/cygwin-announce.mail	1970-01-01 01:00:00.000000000 +0100
+++ CYGWIN-PATCHES/cygwin-announce.mail	2018-07-18 23:52:41.131444600 +0200
@@ -0,0 +1,57 @@
+To: cygwin-announce@cygwin.com
+Subject: Updated: python-PyNaCl 1.2.1 -- Python binding to the Networking and Cryptography (NaCl) library
+--text follows this line--
+
+PACKAGE DESCRIPTION
+===================
+
+Homepage: https://pynacl.readthedocs.io/en/stable/
+License : Apache-2.0
+
+PyNaCl is a Python binding to libsodium, which is a fork of the Networking and
+Cryptography library. These libraries have a stated goal of improving
+usability, security and speed. It supports Python 2.7 and 3.4+ as well as PyPy
+2.6+.
+
+CHANGES SINCE LAST RELEASE
+==========================
+
+- New package.
+
+INSTALL OR UPGRADE NOTES
+========================
+
+Standard install.
+
+CYGWIN INSTALLATION INFORMATION
+===============================
+
+To install this package, click on the "Install Cygwin now" link on the
+<http://cygwin.com/> web page. This downloads setup.exe to your
+system. Then, run setup and answer all of the questions. You'll find
+the package listed in the "All" category. After installation, read the
+documentation at directories:
+
+    /usr/share/doc/<package>/*
+    /usr/share/doc/Cygwin/<package-version>.README
+
+If you have questions or comments, please send them to the Cygwin
+mailing list at <cygwin(at)cygwin.com>.
+
+CYGWIN-ANNOUNCE UNSUBSCRIBE INFO
+================================
+
+This message has been sent to cygwin-announce list.
+
+If you want to unsubscribe from the mailing list, look at the
+"List-Unsubscribe: " tag in the email header of this message. Send
+email to the address specified there. It will be in the format:
+
+    cygwin-announce-unsubscribe-you=yourdomain.com(at)cygwin.com
+
+More information on unsubscribing can be found:
+
+    http://sources.redhat.com/lists.html#unsubscribe-simple
+
+Please read *all* of the information on unsubscribing that is available
+starting at the above URL.
--- CYGWIN-PATCHES/README	1970-01-01 01:00:00.000000000 +0100
+++ CYGWIN-PATCHES/README	2018-07-18 23:52:55.015444600 +0200
@@ -0,0 +1,78 @@
+python-PyNaCl
+------------------------------------------
+
+Python binding to the Networking and Cryptography (NaCl) library
+
+PyNaCl is a Python binding to libsodium, which is a fork of the Networking and
+Cryptography library. These libraries have a stated goal of improving
+usability, security and speed. It supports Python 2.7 and 3.4+ as well as PyPy
+2.6+.+
+
+Runtime requirements:
+  cygwin
+  python2 or python3
+  libsodium23
+
+Build requirements:
+  cygwin
+  cygport
+  python2
+  python2-devel
+  python3
+  python3-devel
+  libsodium-devel
+
+Canonical homepage:
+  https://github.com/pyca/pynacl
+  https://packages.debian.org/python-nacl
+
+Canonical download:
+  See above
+
+Version control repository:
+  git clone https://github.com/pyca/pynacl.git
+
+
+Mailing List:
+
+Forum / Wiki:
+
+IRC channel:
+
+Upstream contact:
+  Author: Donald Stufft
+  Bugs: https://github.com/pyca/pynacl/issues
+
+License:
+  Apache-2.0
+
+Language:
+  Python
+
+------------------------------------
+
+Build instructions:
+  unpack python-PyNaCl-1.2.1-1-src.tar.bz2
+    if you use setup to install this src package, it will be
+    unpacked under /usr/src automatically
+  cd /usr/src
+  cygport python-PyNaCl-1.2.1-1-src/python-PyNaCl.cygport all
+
+This will create:
+  python-PyNaCl-1.2.1-1.tar.xz
+  python-PyNaCl-1.2.1-1-src.tar.xz
+
+-------------------------------------------
+
+Files included in the binary distro:
+  See Cygwin package archive
+
+------------------
+
+Port Notes:
+
+----- version 1.5.2-1 -----
+- Initial release 2018-07-18 Michael Wild
+
+Cygwin port maintained by: Michael Wild <themiwi@gmail.com>
+Copyright (C) 2018 Michael Wild; Licenced under MIT
